gadget/gadget.go

```diff
@@ -462,7 +462,11 @@ func InfoFromGadgetYaml(gadgetYaml []byte, model Model) (*Info, error) {
 	// basic validation
 	var bootloadersFound int
 	knownFsLabelsPerVolume := make(map[string]map[string]bool, len(gi.Volumes))
-	for name, v := range gi.Volumes {
+	for name := range gi.Volumes {
+		v := gi.Volumes[name]
+		if v == nil {
+			return nil, fmt.Errorf("cannot use empty volume %q", name)
```

Comment:
empty volume %q is invalid ?

Reply:
empty volume %q stanza is invalid ?  even

Reply:
I would just say "volume %q stanza is empty", or one might think that there are empty stanzas which can possibly be valid :-)

Reply:
I like the `volume %q stanza is empty` (the rational makes sense to me) the most but no strong preferences, happy to go either way.